### PR TITLE
coverty: addressing unchecked return values

### PR DESCRIPTION
**Pull request by gautvenk**
Signed-off-by: Gautam Venkataramanan <gautam.chennai@gmail.com>

### CODE REVIEW

File: agent-ovs/ovs/SwitchManager.cpp [hunk 1]
```diff
@@ -54,7 +54,8 @@ void SwitchManager::start(const std::string& swName) {
 
 void SwitchManager::connect() {
     connection->RegisterOnConnectListener(this);
-    connection->Connect(OFP13_VERSION);
+    if (!connection->Connect(OFP13_VERSION))
+        LOG(ERROR) << "unable to connect with ovs";
```
Comment (shastrinator) [inline agent-ovs/ovs/SwitchManager.cpp:58]: There is already another error being printed in this path. 
Try this : (void) connection->Connect(OFP13_VERSION);

Comment (gautvenk) [reply agent-ovs/ovs/SwitchManager.cpp]: oh okay. Have removed this error message now.

File: agent-ovs/ovs/SwitchManager.cpp [hunk 2]
```diff
@@ -54,8 +54,7 @@ void SwitchManager::start(const std::string& swName) {
 
 void SwitchManager::connect() {
     connection->RegisterOnConnectListener(this);
-    if (!connection->Connect(OFP13_VERSION))
-        LOG(ERROR) << "unable to connect with ovs";
+    (void)(!connection->Connect(OFP13_VERSION));
```
Comment (shastrinator) [inline agent-ovs/ovs/SwitchManager.cpp:57]: Negation has no effect, since value is being ignored. You can fix and commit it.

Comment (gautvenk) [reply agent-ovs/ovs/SwitchManager.cpp:57]: oops, wasnt intentional. late night i guess:).

**Pull request merged**